sqs.tf

```diff
@@ -1,5 +1,3 @@
-# ignoring because SSE encryption is enabled by default
-# tfsec:ignore:aws-sqs-enable-queue-encryption
```

Comment:
I was able to remove this since we moved from SSE to KMS encryption strategy which is not toggle-able like the SSE option used to be.